src/seismometer/report/auditing.py

```diff
@@ -74,8 +76,7 @@ def fairness_audit_to_html(
     df.drop(score_column, axis=1, inplace=True)
 
     if df["score"].nunique() != 2:
-        logger.error("Audit requires exactly two target classes to be present")
-        return
+        raise CensoredResultException("Audit requires exactly two target classes to be present")
```

Comment:
Q: From the changelog, a CensoredResultException is "when a plot cannot render due to too few results". Doublechecking that this fits that description?

Reply:
Yeah, there are too few results that are in the positive class (or visa versa, its all positive)